app/javascript/data/advancedFilterOptions.js

```diff
@@ -340,6 +305,67 @@ export const advancedFilterOptions = [
   },
 
   /* COMBINATION OPTIONS */
+  {
+    name: 'address',
+    title: 'Address (Combination)',
+    description: 'Monitorees with specified address',
+    type: 'combination',
+    fields: [
+      {
+        name: 'address-foreign',
+        title: 'outside USA',
+        description: 'Monitoree Address 1, Town/City, Country, Address 2, Postal Code, Address 3 or State/Province (outside USA)',
+        type: 'search',
+      },
+      {
+        name: 'address-usa',
+        title: 'within USA',
+        description: 'Monitoree Address 1, Town/City, State, Address 2, Zip, or County within USA',
+        type: 'search',
+      },
+    ],
+  },
+  {
```

Comment:
Does it make sense to switch the ordering of 'outside USA' and 'within USA'? (I know it's not alphabetical, but might it correspond to usage patterns if 'within USA' came first?)

Reply:
good question, @sgober @tstrass any thoughts?

Reply:
My instinct is to swap them but curious what @hackrm thinks. Could also do "Domestic" and "International"

Reply:
I agree with swapping them and having "within USA" come first.

And while I would like to swap out the current language for "domestic" and "international", since we use the within/outside terminology in the Edit Address page of a monitoree, I believe it is best to remain consistent with it and stick with within/outside. 

![Screen Shot 2022-01-10 at 5 00 27 PM](https://user-images.githubusercontent.com/42656458/148846178-85d81d08-4363-49ce-9d72-1a00783eb28f.png)

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1179/commits/e83da2878fe36c79f0563fd020c4762ba8e8c98f